pkg/compiler/codegen.go

```diff
@@ -1347,13 +1347,13 @@ func (c *codegen) isCallExprSyscall(e ast.Expr) bool {
 // 1. `defer` is always executed irregardless of whether an exception has occurred.
 // 2. `recover` can or can not handle a possible exception.
```

Comment:
Why these are not lists, btw?

Reply:
They are not fixed by `gofmt` automatically because they don't have inconsistent line breaks. So it's OK from the gofmt side to have it as not-list. Fixed manually.

Reply:
I guess we have a lot of cases like that then. Well, they'll be improved eventually.